app/services/address_lookup_service.rb

```diff
@@ -2,6 +2,7 @@ class AddressLookupService
   prepend SimpleCommand
 
   ORDNANCE_SURVEY_URL = 'https://api.ordnancesurvey.co.uk/places/v1/addresses/postcode'.freeze
+  POSTCODE_REGEXP = /\A([A-Z][A-HJ-Y]?[0-9][A-Z0-9]?[0-9][A-Z]{2}|GIR ?0A{2})\z/.freeze
```

Comment:
Does this need to be duplicated?  There is already a version in settings, can that be referenced?